docs/src/model_creation/examples/basic_CRN_library.md

```diff
@@ -1,4 +1,22 @@
 # [Library of Basic Chemical Reaction Network Models](@id basic_CRN_library)
+```@raw html
+<details><summary><strong>Tutorial environment set-up</strong></summary>
+```
+The following code sets up an environment for running this tutorial.
+```julia
+using Pkg
+Pkg.activate(".")
+Pkg.add("Catalyst")
+Pkg.add("OrdinaryDiffEqDefault")
+Pkg.add("StochasticDiffEq")
+Pkg.add("JumpProcesses")
+Pkg.add("Plots")
+```
+```@raw html
+</details>
+```
+  \
```

Comment:
The "   \" is kind of ugly, but is required to add some space between the expandable menu and the actual text. There might be a markdown command which is more clear that that is what it does though.